deploy/apecloud-mysql/templates/clusterdefinition.yaml

```diff
@@ -68,19 +68,25 @@ spec:
         backupPolicyTemplateName: apecloud-mysql-backup-policy-for-hscale
       switchoverSpec:
         withCandidate:
-          image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ default .Values.image.tag }}
-          command:
-            - mysql
-          args:
-            - -h$(KB_CONSENSUS_LEADER_POD_FQDN)
-            - -u$(MYSQL_ROOT_USER)
-            - -p$(MYSQL_ROOT_PASSWORD)
-            - -e
-            - "call dbms_consensus.change_leader('$(KB_SWITCHOVER_CANDIDATE_FQDN):13306');"
+          cmdExecutorConfig:
+            image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ default .Values.image.tag }}
+            command:
+              - mysql
+            args:
+              - -h$(KB_CONSENSUS_LEADER_POD_FQDN)
+              - -u$(MYSQL_ROOT_USER)
+              - -p$(MYSQL_ROOT_PASSWORD)
+              - -e
+              - "call dbms_consensus.change_leader('$(KB_SWITCHOVER_CANDIDATE_FQDN):13306');"
+          configMapRefs:
+            - name: apecloud-mysql-scripts
```

Comment:
how is mount path specified?

Reply:
if there are a number of keys (as files) in a configmap  what will happen if key/filename is conflicted?

Reply:
> if there are a number of keys (as files) in a configmap what will happen if key/filename is conflicted another configmap?

This field has been changed to scriptSpecSelector, which is used to reference the scripts in scriptSpecs. There is no distinction made for scenarios where there may be key conflicts. If there are duplicate names in scriptSpecs, all of them will be loaded and referenced.